bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss

```diff
@@ -31,8 +31,32 @@
   }
 
   &.glow {
-    span:first-child{
-      box-shadow: 0 0 0 1px #ffffff, 0 0 0 1px;
-    }
+    -webkit-animation: pulse 1s infinite ease-in-out;
+    animation: pulse 1s infinite ease-in-out;
+    border-radius: 4em;
+  }
+}
+
+@-webkit-keyframes pulse {
```

Comment:
You don't need this, when scss is compiled, the compiler adds the prefix. 

![webkit](https://user-images.githubusercontent.com/15806883/40935381-52edb1e4-680e-11e8-97e0-5528bd6e7f71.png)